package.json

```diff
@@ -45,12 +45,14 @@
   },
   "dependencies": {
     "@radix-ui/react-dialog": "^0.1.7",
+    "classnames": "^2.3.2",
```

Comment:
Let's create a utility for this, rather than import a whole dependency:
```ts
/**
 * CSS classnames utility that allows you to compose multiple classes and turn
 * them into a string.
 */
export function cx(...args: Array<string | undefined | false>) {
  return args.filter((arg) => arg && typeof arg === "string").join(" ");
}
```

Reply:
This has other uses though, such as `classNames("some classes", { hidden: isHidden() }`), which I actually use in this PR.

I've read that it's a very popular package that some people automatically add in most of their projects, so I thought it would be useful (plus it seems very small).

You're the front-end specialist, so your call :)

Reply:
Personally I'd use a utility, because it's small and flexible. The example you gave could be written like this:
```ts
cx("some classes", isHidden() && "hidden");
```

However, I also realize that these are minor improvements, which we don't have to spend a lot of time on. We have bigger "Tofish" to fry 😄